docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# CONTRIBUTING
+
+> Before contributing, we highly recommend introducing yourself and your ideas on gitter.[![Gitter](https://badges.gitter.im/codEd-org/simulate.svg)](https://gitter.im/codEd-org/simulate?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+### Setting Up Simulate:
+
+1. Make sure you have flutter installed (https://flutter.dev/docs/get-started/install)
+2. Fork the repository
+3. Clone the forked repository:
+    ```console
+    git clone https://github.com/[YOUR USERNAME]/simulate.git
+    ```
```

Comment:
If you're using `https`,
```console
git clone https://github.com/[YOUR USERNAME]/simulate.git
```
If you're using `ssh`,
```console
git clone git@github.com:[YOUR USERNAME]/simulate.git
```

Reply:
I am going with the default thing